ios/Sources/HomeFeature/HomeScreen.swift

```diff
@@ -65,29 +61,55 @@ public struct HomeScreen: View {
     }
 }
 
+private extension HomeAction {
+    init(action: HomeScreen.ViewAction) {
+        switch action {
+        case .progressViewAppeared:
+            self = .refresh
+        }
+    }
+
+    init(action: HomeContentAction) {
+        self = .homeContent(action)
+    }
+}
+
+#if DEBUG
```

Comment:
previews works in debug configuration. you don't need to wrap this.

Reply:
This `#if` includes mock extensions too.
The mock extensions are included in not debug configuration. So I added `#if`.

Reply:
I think this is okay for this problem
https://github.com/DroidKaigi/conference-app-2021/pull/528#issuecomment-874223853